greenheart/tools/eco/finance.py

```diff
@@ -369,12 +467,12 @@ def run_profast_lcoe(
         },
     )
     pf.set_params(
-        "capacity", hopp_results["annual_energies"]["wind"] / 365.0
+        "capacity", hopp_results["annual_energies"]["hybrid"] / 365.0
```

Comment:
` hopp_results["annual_energies"]["hybrid"]` is limited by the interconnection size, so this can be a problem if the "interconnection size" is too small it will curtail the "energy to the grid" (even though it's not technically going to the grid, that's how PySAM handles it). So if you sum the individual power profiles from each technology you will get a different value.

Reply:
I changed this to base on the pre_interconnect_kwac value that is being provided to power the rest of the ECO